packaging/check-for-go-toolchain.sh

```diff
@@ -16,6 +16,7 @@
 GOLANG_MIN_MAJOR_VERSION='1'
 GOLANG_MIN_MINOR_VERSION='21'
 GOLANG_MIN_PATCH_VERSION='0'
+GOLANG_MIN_VERSION="${GOLANG_MIN_MAJOR_VERSION}.${GOLANG_MIN_MINOR_VERSION}.${GOLANG_MIN_PATCH_VERSION}"
```

Comment:
@Ferroin ,

Please, take a look in next warning.

Reply:
This gets used by the installer script, which sources this file.